packages/core/src/textures/resources/BufferResource.js

```diff
@@ -1,5 +1,9 @@
 import Resource from './Resource';
 
+/**
+ * @interface SharedArrayBuffer
+ */
```

Comment:
Is this needed?

Reply:
Yes, unfortunately we cannot ask users to add es2017 lib to their tsconfig yet.